concrete-ml/svm.ipynb

```diff
@@ -0,0 +1,450 @@
+    "\n",
+    "In order to perform prediction in a FHE environment, the model first has to be compiled into a circuit.\n",
+    "\n",
+    "#### Third, the model is compiled to enable FHE execution\n",
+    "\n",
+    "```python\n",
+    "# A circuit needs to be compiled to enable FHE execution\n",
+    "circuit = svm_concrete.compile(X_train)\n",
+    "circuit.client.keygen(force=False)\n",
+    "# Now that a circuit is compiled, the svm_concrete can predict value with FHE\n",
+    "y_pred_concrete_fhe = svm_concrete.predict(X_test, execute_in_fhe=True)\n",
+    "accuracy_concrete_fhe = accuracy_score(y_test, y_pred_concrete_fhe)\n",
+    "```\n",
+    "\n",
+    "Now that the model is compiled, computing predictions in with FHE is just a matter of passing a execution_in_fhe parameter set to True.\n",
+    "\n",
+    "\n",
+    "#### Accuracy\n",
+    "\n",
+    "Finally, we can measure the accuracy of our 3 different predictions:\n",
+    "\n",
+    "```python\n",
+    "print(f\"Scikit-learn Accuracy: {accuracy_sklearn:.4f}\")\n",
+    "print(f\"Concrete-ML Clear Accuracy: {accuracy_concrete_clear:.4f}\")\n",
+    "print(f\"Concrete-ML FHE Accuracy: {accuracy_concrete_fhe:.4f}\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51afbfe2",
+   "metadata": {},
+   "source": [
+    "### Key takeaways\n",
+    "\n",
+    "#### Simplicity of execution\n",
+    "\n",
+    "For a high-level use-case, Concrete-ML offers a very similar interface to scikit-learn. The main difference is *a model needs to be compiled to allow execution in FHE*.\n",
+    "\n",
+    "#### Model Accuracy\n",
+    "\n",
+    "Concrete-ML prediction accuracy is slightly under scikit-learn implementation. This is because of [quantization](https://docs.zama.ai/concrete-ml/advanced-topics/quantization): number precision needs to be fixed-size for the model to be evaluated in FHE. This can be alleviated down to where the accuracy difference is none or negligible.\n",
+    "\n",
+    "#### Execution time\n",
+    "\n",
+    "The execution is slower with Concrete-ML, especially when compiling the model. Enabling and using encryption indeed uses a lot more resources than training and using a model using plain data. The speed can be increased by *reducing the precision of the data* (understand diminish the fixed-size number precision). Depending on the project, you thus have to choose between:\n",
+    "\n",
+    "- a slower model that performs accurate predictions\n",
+    "- a faster model that performs less accurate predictions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdc4c4d2",
+   "metadata": {},
+   "source": [
+    "## Part 2: In-Depth model development\n",
+    "\n",
+    "a more in-depth approach, showing how to effectively develop with concrete-ml. This will quote and follow the steps of [model development](https://docs.zama.ai/concrete-ml/getting-started/concepts#i.-model-development)\n",
```

Comment:
```suggestion
    "A more in-depth approach, showing how to effectively develop with concrete-ml. This will quote and follow the steps of [model development](https://docs.zama.ai/concrete-ml/getting-started/concepts#i.-model-development)\n",
```